### DISCUSSION TITLE
CMake Configure File

### DISCUSSION THREAD

**Discussion by dariomangoni** [InteractiveComputerGraphics/SPlisHSPlasH#293] [Ideas]
Hi, is there a specific reason why **there isn't any configure file** to allow an easier linking from other tools through CMake?
Given the many linked libraries it takes some time to figure out all the dependencies, compiler options, defines, etc... and recreate them in an outside project. It would surely be easier with a config file.
From your CMakeLists I can see that you are surely proficient in the use of CMake, so it's surely not a problem of being able to!

Did you simply considered SPlisHSPlasH to be an end-tool thus not justifying the need of it?
Would you be interested in a contribution?

Thanks,
Dario

**Comment by janbender** [comment]
The answer is simply that no one had time yet to write a configuration file ;-)

**Comment by dariomangoni** [comment]
Great, if I come up with some solution I will propose it to you.
